src/main/java/crawlercommons/sitemaps/extension/ImageAttributes.java

```diff
@@ -128,4 +128,29 @@ public boolean equals(Object other) {
                         && Objects.equals(license, that.license);
     }
 
+    @Override
+    public Map<String, String[]> asMap() {
+        Map<String, String[]> map = new HashMap<>();
+
+        if (loc != null) {
+            map.put("loc", new String[]{ loc.toString() });
```

Comment:
For this, and all other xxxAttributes classes, are there well defined map key name constants we could use? 

If not, then I would be in favor of adding them as public constants to the xxxAttribute classes, to avoid raw string constant usage in consumer code.

Reply:
There are not constants that I could fine.  I'll add some and use those instead of strings.